microprofile.jdt/com.redhat.microprofile.jdt.test/src/main/java/com/redhat/microprofile/jdt/core/BasePropertiesManagerTest.java

```diff
@@ -145,8 +168,8 @@ public void run(IProgressMonitor monitor) throws CoreException {
 		return javaProject;
 	}
 
-	private static File copyProjectToWorkingDirectory(String projectName) throws IOException {
-		File from = new File("projects/maven/" + projectName);
+	private static File copyProjectToWorkingDirectory(String projectName,  String parentDirName) throws IOException {
+		File from = new File("projects/" + parentDirName+ "/" + projectName);
 		File to = new File(JavaUtils.getWorkingProjectDirectory(), projectName);

```

Comment:
As we have gradle and maven project which could have the same name, we should set the parentDirName in the getWorkingProjectDirectory.